docs/contributing.rst

```diff
@@ -57,7 +57,7 @@ In order to use the hook as defined in the pre-commit-config.yaml run:
 
 In order to install the package for development, install in editable mode with optional dependencies.
 
-See axiomapy\examples\with_entitybase examples for info on how new instances should be defined.
+See axioma-py\\examples\\with_entitybase examples for info on how new instances should be defined.
```

Comment:
a qu?estion: should we make it forward slash so that it is compatible for both windows and linux